src/coreclr/tools/aot/ILCompiler.LLVM/CodeGen/ILToLLVMImporter.cs

```diff
@@ -17,6 +17,7 @@
 using ILCompiler.DependencyAnalysisFramework;
 using ILCompiler.LLVM;
 using Internal.IL.Stubs;
+using Internal.ReadyToRunConstants;
```

Comment:
Is this needed?

Reply:
No, removed (any a few others that weren't needed).